data/json/items/tool/science.json

```diff
@@ -364,13 +364,16 @@
     "symbol": "U",
     "color": "dark_gray",
     "looks_like": "household_water_heater",
-    "weight": "22700 g",
+    "weight": "500 kg",
```

Comment:
At this point, shouldn't it become an appliance? Especially since you made it take large storage batteries.

Reply:
I'm pretty sure someone else is doing that. #64830
hopefully this won't make a conflict for em lol.